### ISSUE TITLE
Empty comment are considered as error

### ISSUE THREAD

**Issue by Fymyte** [amaanq/tree-sitter-kdl#7]
Hi, thanks for this parser :+1: 

Comments with only `//` are considered as error:
```kdl
// Examples:
//
//something
// 
```
(There is a space after the last `//`) tree:
```scm
(single_line_comment)
(ERROR)
(single_line_comment)
(single_line_comment)
```

**Comment by amaanq** [comment]
Thank you for the bug report, I'll investigate now

**Comment by amaanq** [comment]
Fixed in v1.0.3

**Comment by Fymyte** [comment]
Too quick thx 😀
